Gems/WhiteBox/Code/Source/SubComponentModes/EditorWhiteBoxTransformMode.cpp

```diff
@@ -125,6 +134,139 @@ namespace WhiteBox
             m_transformClusterId);
 
         DestroyManipulators();
+
+        EditorWhiteBoxTransformModeRequestBus::Handler::BusDisconnect();
+    }
+
+    void TransformMode::RegisterActionUpdaters()
+    {
+    }
+
+    void TransformMode::RegisterActions()
+    {
+        auto actionManagerInterface = AZ::Interface<AzToolsFramework::ActionManagerInterface>::Get();
+        AZ_Assert(actionManagerInterface, "WhiteBoxTransformMode - could not get ActionManagerInterface on RegisterActions.");
+
+        auto hotKeyManagerInterface = AZ::Interface<AzToolsFramework::HotKeyManagerInterface>::Get();
+        AZ_Assert(hotKeyManagerInterface, "WhiteBoxTransformMode - could not get HotKeyManagerInterface on RegisterActions.");
+
+        // Translation
+        {
+            constexpr AZStd::string_view actionIdentifier = "o3de.action.whiteBoxComponentMode.transform.translation";
+            AzToolsFramework::ActionProperties actionProperties;
+            actionProperties.m_name = SwitchToTranslationModeTile;
+            actionProperties.m_description = SwitchToTranslationModeDesc;
+            actionProperties.m_category = "White Box Component Mode - Transform";
+
+            actionManagerInterface->RegisterAction(
+                EditorMainWindowActionContextIdentifier,
+                actionIdentifier,
+                actionProperties,
+                [&]
```

Comment:
update: Won't mention again but same comment about this likely being able to switch to `[this]` and any other values that need to be captured explicitly

Reply:
These were unnecessary again, removed.